src/static/css/main.less

```diff
@@ -81,6 +81,16 @@ body {
     overflow-wrap: break-word;
 }
 
+blockquote {
+
+  border-left: 0.25em solid @gray;
+  color: @dark-gray;
+  padding: 0.5em 1em 0.25em 1em;
```

Comment:
These are all just temporary styles to do something more than basic indentation until we properly define a style for `blockquote`, right?